const test = require('blue-tape')
const {ConfigurationValidator} = require('..')

test('Throws an instance of Error', async (t) => {
  const validator = new ConfigurationValidator()
  const given = {foo: 'bar'}
  t.plan(1)
  try {
    validator.validate(given)
  } catch (error) {
    t.true(error instanceof Error)
  }
})
